shaft_element

```diff
@@ -0,0 +1,478 @@
+import warnings
+import numpy as np
+import pandas as pd
+import scipy.interpolate as interpolate
+import matplotlib.pyplot as plt
+import matplotlib.patches as mpatches
+from ross.data_io.read_xl import (
+    load_bearing_seals_from_yaml,
+    load_bearing_seals_from_xltrc,
+    load_disks_from_xltrc,
+    load_shaft_from_xltrc,
+)
+
+"""The path "ross.data_io.read_x1" on line 7 maybe changes"""
+
+
+__all__ = [
+    "ShaftElement"
+]
+
+
+class Element:
```

Comment:
Implement `class Element` as `ABC`. 
The docs for ABC can be found [here](https://docs.python.org/3/library/abc.html) and an interesting article [here](https://dbader.org/blog/abstract-base-classes-in-python).